src/test/java/org/wmn4j/io/musicxml/MusicXmlFileChecks.java

```diff
@@ -0,0 +1,713 @@
+package org.wmn4j.io.musicxml;
+
+import org.wmn4j.notation.elements.*;
```

Comment:
It is generally better to avoid imports that bring in more things than needed. These wildcard imports could be changed to just import the specific classes that are used.

Reply:
Looks like I had an automatic setting on in my IntelliJ that grouped imports to a wildcard import if too many classes were imported from the same package :stuck_out_tongue: